kaniko-example/README.md

```diff
@@ -30,7 +30,11 @@ This is an example showing how to collect build-info, while using the Kaniko con
 
     ```console
     Run Kaniko:
-    > docker run --rm -v `pwd`:/workspace -v `pwd`/config.json:/kaniko/.docker/config.json:ro gcr.io/kaniko-project/executor:latest --dockerfile=Dockerfile --destination=DOCKER_REG_URL/hello-world:latest --image-name-with-digest-file=image-file-details
+    > docker run --rm -v `pwd`:/workspace -v `pwd`/config.json:/kaniko/.docker/config.json:ro gcr.io/kaniko-project/executor:$(IMAGE_TAG) --dockerfile=Dockerfile --destination=DOCKER_REG_URL/hello-world:latest --image-name-with-digest-file=image-file-details
+
+    Add image tag:
+    // If you are running on OSX, replace `-i` with `-i ''`:
+    > sed  -i 's/@/:$(IMAGE_TAG) @/g' image-file-details
```

Comment:
I suggest replacing this section:
```
    Add image tag:
    // If you are running on OSX, replace `-i` with `-i ''`:
    > sed  -i 's/@/:$(IMAGE_TAG) @/g' image-file-details
```
with the following:
```
    Add the image tag to 'image-name-with-digest-file':
    > sed  -i 's/@/:$(IMAGE_TAG) @/g' image-file-details
    On macOS, use:
    > sed  -i '' 's/@/:$(IMAGE_TAG) @/g' image-file-details
```

Reply:
I think `Add the image tag to 'image-name-with-digest-file'` is wrong file name. 
I suggest `Add the image tag to 'image-file-details'`